src/main/java/tools/jackson/core/TreeNode.java

```diff
@@ -227,7 +228,7 @@ public interface TreeNode
      * @return An iterator for traversing names of all properties this Object node
      *   has (if Object node); empty {@link Iterator} otherwise (never {@code null}).
      */
```

Comment:
@cowtowncoder the javadoc is out of date (mentions Iterator). Can the new the new javadoc mention if the collection returned is the live version of the data or a safe copy?

If this no longer returns iterator, then we probably don't need the new spliterator method either. Users can get their own iterator or spliterator from the collection.

Reply:
Thanks will update. Live-ness is up to implementation tho, `TreeNode` does not control that aspect.